app.js

```diff
@@ -1,7 +1,13 @@
-const express = require('express')
-const app = express()
-const port = 3000
+module.exports = (/*options*/) => {
+  // Use options.server to access http.Server. Example with socket.io:
+  //     const io = require('socket.io')(options.server)
+  const app = require('express')()
 
-app.get('/', (req, res) => res.send('Hello World!'))
+  app.get('/', (req, res) => {
+    // Use req.log (a `pino` instance) to log JSON:
+    req.log.info({message: 'Hello from Appsody!'});
```

Comment:
I liked carlos's suggestion of making this log the metrics, readiness and liveness endpoints.  Not sure if it fits in the model given that they are added by the infra and this is an example app, but thought I'd mention to see if you'd considered that.

Reply:
I could, from a user-perspective, it sounds nice, but it I am concerned that doing it would be demonstrating anti-patterns in the sample app:

1. there are good ways to send html to the client, and bad ways, and if we just have a hard-coded html string we dump we are doing it the bad way, but doing the better ways are all bringing a lot of machinery for html generation into a simple application that doesn't otherwise need it

2. the app isn't supposed to know anything about the stack-defined routes around it, so once again, though its convenient, having the app generate an HTML page that refers to routes back in the stack seems an anti-pattern in that it breaks seperation of concerns

Also, I think its something that should be done across the stacks/CP4Apps, not just a per-stack one-off.

That said, if the benefits of a nicer HTML welcome page are compelling, I could do it.

WDYT?